.github/workflows/ci.yml

```diff
@@ -40,3 +41,14 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+      - name: Run coverage
+        if: success() && matrix.os == 'ubuntu-latest' && matrix.node == '12.x'
```

Comment:
I don't know if `success()` is correct here
What if some tests are failing? Is the report not running?
Is that intended?

Reply:
Removed success